proc/proc_test.go

```diff
@@ -676,6 +676,11 @@ func TestCGONext(t *testing.T) {
 	if runtime.GOOS == "darwin" && strings.Contains(runtime.Version(), "1.4") {
 		return
 	}
+	
+	if runtime.GOOS == "darwin" {
+		//XXX- temporarily disabled due to inability to read DWARF info on macOS >= 10.12.1
```

Comment:
I won't block on merging because of this, but let's create an issue so we can do a follow up PR.